Userland/Libraries/LibJS/Bytecode/Op.cpp

```diff
@@ -496,6 +496,25 @@ ThrowCompletionOr<void> JumpUndefined::execute_impl(Bytecode::Interpreter& inter
     return {};
 }
 
+static ThrowCompletionOr<MarkedVector<Value>> argument_list_evaluation(Bytecode::Interpreter& interpreter)
+{
+    auto& vm = interpreter.vm();
+
+    MarkedVector<Value> argument_values { vm.heap() };
+    auto argument_array = interpreter.accumulator();
+
+    // No Arguments
+    if (argument_array.is_undefined())
+        return argument_values;
+
+    TRY(get_iterator_values(vm, argument_array, [&argument_values](Value val) -> Optional<Completion> {
+        argument_values.append(val);
+        return {};
+    }));
```

Comment:
Nope, we're not doing this. Please check what `get_iterator_values` actually does and then tell my why this is a terrible idea (hint: it involves calling into user code)